specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json

```diff
@@ -300,6 +300,88 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MachineLearningServices/workspaces/{resourceName}/listresourcepreparationerrors": {
+      "post": {
+        "tags": [
+          "ProxyOperations"
+        ],
+        "operationId": "ListResourcePreparationErrors",
+        "consumes": [],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
```

Comment:
Are these options here intentional to override the global settings [here](https://github.com/Azure/azure-rest-api-specs/blob/39e2bfa457e58e57efed3fbbce5ffba87ac24a12/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-03-01/machineLearningServices.json#L12)? Or is this valid?

Reply:
Sorry, I should remove them...